documentation/modules/operators/con-deleting-managed-topics.adoc

```diff
@@ -56,4 +56,31 @@ When the finalization tasks are successfully executed, the finalizer is removed
 
 Finalizers also prevent related resources from being deleted. 
 If the unidirectional Topic Operator is not running, it won't be able to remove the `metadata.finalizer`. 
-Consequently, an attempt to delete the namespace that contains the `KafkaTopic` resource won't complete until either the operator is restarted, or the finalizer is otherwise removed (for example using `kubectl edit`). 
\ No newline at end of file
+Consequently, an attempt to delete the namespace that contains the `KafkaTopic` resource won't complete until either the operator is restarted, or the finalizer is otherwise removed (for example using `kubectl edit`). 
+
+== Removing finalizers
+
+If you want to bypass the finalization process when deleting topics, you can remove the finalizers.   
+You might want to do this to delete topics more quickly; or to handle certain topics without invoking cleanup operations.
+Perhaps the cleanup operation associated with the finalization process is not completing as expected.
```

Comment:
I would probably say something like this:

```suggestion
In order to delete the `KafkaTopic` resource while the Topic Operator is not running, you have to remove the finalizers first.
You can do it manually by editing the resources.
Or you can use one of the example commands below.
```

I think we should stick with the _when the TO is not running_ and not try to explain the situations when that might happen as there might be many different situations.

Reply:
Perhaps rather than `you have to remove the finalizers first.` we should reword it like this: `you have to remove the finalizers yourself`, because IIRC it's fine to delete the namespace first, then delete the finalizer (i.e. 'first' is incorrect).